extensions/config-yaml/runtime/src/main/java/io/quarkus/yaml/configuration/runtime/FileSystemYamlConfigSource.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.yaml.configuration.runtime;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public abstract class FileSystemYamlConfigSource extends YamlConfigSource {
+
+    private static final long serialVersionUID = -3881751839186883952L;
+
+    protected FileSystemYamlConfigSource(String source, int priority) {
+        super(source, priority);
+    }
+
+    @Override
+    protected InputStream openYamlFileStream() {
+        Path path = Paths.get("config", source);
```

Comment:
So this always assumes that a `config/<file>` exists in the project?

Reply:
No, building the `Path` doesn't require the file to exist.